_config.yml

```diff
@@ -11,8 +11,8 @@ description: > # this means to ignore newlines until "baseurl:"
   Write an awesome description for your new site here. You can edit this
   line in _config.yml. It will appear in your document head meta (for
   Google search results) and in your feed.xml site description.
-baseurl: "" # the subpath of your site, e.g. /blog
-url: "http://yourdomain.com" # the base hostname & protocol for your site
+baseurl: "/amp" # the subpath of your site, e.g. /blog
+url: "https://zakibinary.github.io/amp-landing-pages" # the base hostname & protocol for your site
```

Comment:
We should change this to production subdomain which we set later. let me see what subdomain we'd like to use.